mllib/src/main/scala/org/apache/spark/ml/tree/treeParams.scala

```diff
@@ -360,7 +362,8 @@ private[ml] trait RandomForestParams extends TreeEnsembleParams {
     "The number of features to consider for splits at each tree node." +
       s" Supported options: ${RandomForestParams.supportedFeatureSubsetStrategies.mkString(", ")}",
     (value: String) =>
-      RandomForestParams.supportedFeatureSubsetStrategies.contains(value.toLowerCase))
+      RandomForestParams.supportedFeatureSubsetStrategies.contains(value.toLowerCase)
+      || value.matches("^(?:[1-9]\\d*|0\\.\\d*[1-9]\\d*\\d*|1\\.0+)$"))
```

Comment:
Thanks @MLnick @sethah. In `0\\.\\d*[1-9]\\d*\\d*` the last `\\d*` is to capture situations of trailing zero `"0.0250"`. I didn't take into consideration of `".25"` case initially.

Let me update the pull request and address those issues.